packages/freecad.rb

```diff
@@ -9,16 +9,16 @@ class Freecad < Package
   source_url 'https://github.com/FreeCAD/FreeCAD/releases/download/0.19.2/FreeCAD_0.19-24291-Linux-Conda_glibc2.12-x86_64.AppImage'
   source_sha256 'c196a6e59349ed452cc9b9af2c6a0d983a698831630aa5c7077565ed8570c9ad'
 
-  binary_url ({
+  binary_url({
   })
-  binary_sha256 ({
+  binary_sha256({
   })
 
   depends_on 'sommelier'
 
   def self.preflight
     free_space = `echo $(($(stat -f --format="%a*%S" .)))`.chomp.to_i
-    abort 'Not enough free disk space.  You need at least 5.2 GB to install.'.lightred if free_space < 5583457485
+    abort 'Not enough free disk space.  You need at least 5.2 GB to install.'.lightred if free_space < 5_583_457_485
```

Comment:
This is wrong.  Should be a number.

Reply:
@Zopolis4 Can you get this taken care of?